lib/services/trafficManagerManagement2/README.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure SDK for Node.js - Traffic Manager Management
+
+This project provides a Node.js package that makes it easy to manage Microsoft Azure Traffic Manager. 
```

Comment:
Please consider add a "Resources" at the end, so that people know this is for ARM. Like "manage Microsoft Azure Traffic Manager Resources."